pkg/systemd/services/service-details.jsx

```diff
@@ -208,11 +208,40 @@ export class ServiceDetails extends React.Component {
             waitsFileAction: false,
             note: "",
             error: "",
+            delay: 30000, // 30s
```

Comment:
This is a bit weird as state -- should rather be a const above the class? `METRICS_POLL_DELAY` or so?

Reply:
Done